### PR TITLE
Add profiles that should be shared with all pipelines to configs

### PR DESCRIPTION
**Pull request by MaxUlysse**
I do think `test` and `debug` should be the only profiles to be specified at the pipeline level.

So I'd like to include the following profile here:
- awsbatch

I still think `conda`, `docker` and `singularity` should be included there, but I agree that we can do that one first, and then we'll see.

### CODE REVIEW

File: conf/awsbatch.config [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//Nextflow config file for running on AWS batch
+
+params {
+  config_profile_name = 'AWSBATCH'
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Alexander Peltzer (@apeltzer)'
+  config_profile_url = 'https://aws.amazon.com/batch/'
+}
+
+aws.region = params.awsregion
```
Comment (ewels) [inline conf/awsbatch.config]: This line causes the tests to fail 😱 

https://travis-ci.com/nf-core/configs/jobs/258237834#L362

File: conf/awsbatch.config [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+process.executor = 'awsbatch'
+process.queue = params.awsqueue
```
Comment (ewels) [inline conf/awsbatch.config]: This line will also cause problems.

File: conf/awsbatch.config [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+executor.awscli = '/home/ec2-user/miniconda/bin/aws'
+params.tracedir = './'
```
Comment (ewels) [inline conf/awsbatch.config]: Any reason not to group this together with the other `params` at the top?

Comment (MaxUlysse) [reply conf/awsbatch.config]: None, I just copied the config file we had on sarek

**Pull request merged**